net-dialup/accel-ppp/files/accel-ppp-1.12.0-tsearch.patch

```diff
@@ -0,0 +1,20 @@
+# Don't type cast l2tp_tunnel_free_sessions on musl libc
+#
+# Taken from Alpine linux patch:
+# https://git.alpinelinux.org/aports/tree/community/accel-ppp/0003-tsearch.patch
+#
+# Closes: https://bugs.gentoo.org/828906
+--- a/accel-pppd/ctrl/l2tp/l2tp.c
++++ b/accel-pppd/ctrl/l2tp/l2tp.c
+@@ -852,7 +852,11 @@ static void l2tp_tunnel_free_sessions(struct l2tp_conn_t *conn)
+ 	void *sessions = conn->sessions;
+
+ 	conn->sessions = NULL;
++#if defined(__GLIBC__)
```

Comment:
Is the cast really needed on glibc? This feels like the kind of thing we should be able to do correctly unconditionally.

Reply:
Yes, it's needed in glibc. Else it throws error

Reply:
Huh, @thesamesam. You're right, it doesn't give any error without the cast on glibc. I thought I previously faced a build error with the cast on glibc.